src/components/GroupSystems/GroupImmutableSystems.js

```diff
@@ -258,13 +261,19 @@ const GroupImmutableSystems = ({ groupName, groupId, ...props }) => {
                     requiredPermissions={REQUIRED_PERMISSIONS_TO_MODIFY_GROUP(
                       groupId
                     )}
-                    noAccessTooltip={NO_MODIFY_GROUP_TOOLTIP_MESSAGE}
+                    noAccessTooltip={
+                      isWorkspaceEnabled
+                        ? NO_MODIFY_WORKSPACE_TOOLTIP_MESSAGE
+                        : NO_MODIFY_GROUP_TOOLTIP_MESSAGE
+                    }
```

Comment:
While this does the job, it looks like this appears a couple of times.

It might make sense to declare this as and maybe also the "remove label" as `const`s above and use these instead:
 
```js
const noAccessTooltip = isWorkspaceEnabled
                         ? NO_MODIFY_WORKSPACE_TOOLTIP_MESSAGE
                         : NO_MODIFY_GROUP_TOOLTIP_MESSAGE;
const removeLabel = isWorkspaceEnabled
                       ? 'Remove from workspace'
                       : 'Remove from group';
```

This would also make the removing of the flag later on easier.